api/cmd/query/main.go

```diff
@@ -57,13 +59,21 @@ func parseopts() opts {
 		"Signing key used for response authorization tokens",
 		"key",
 	)
+	port := getopt.IntLong(
```

Comment:
Why parse as int with getopt, when you want the port as string anyways?

Reply:
Original idea was that port is a number. So lets parse it as a number! 
But if you are wondering I probably better change it to a string. It will fail fast enough anyway.

Reply:
Not a big deal, it's probably gonna panic with a descriptive message in both cases